backend/routes/discord.py

```diff
@@ -79,5 +79,4 @@ async def get(self, request: Request) -> JSONResponse:
 
         if member:
             return JSONResponse(member.dict())
-        else:
-            return NOT_FOUND_EXCEPTION
+        return NOT_FOUND_EXCEPTION
```

Comment:
I think the explicit else is more readable. At the least there should be a blank line after the if block.